app/models/user.rb

```diff
@@ -0,0 +1,29 @@
+class User
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String, required: true
+  property :username, String, required: true, unique: true, message: 'Username is already taken'
```

Comment:
Metrics/LineLength: Line is too long. [96/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)